buildcontext/git.go

```diff
@@ -8,24 +8,24 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/earthly/earthly/conslogging"
+	"github.com/earthly/earthly/cleanup"
```

Comment:
is this rename intentional? are you wanting to circle back to cleanup this package later?